(Unix.Unix_error "Illegal seek" sysconf "") with glibc-2.26

### How to reproduce

With glibc-2.26,

```sh-session
$ git clone --recursive https://github.com/gfngfn/SATySFi.git
$ cd SATySFi
$ opam pin add satysfi .
$ opam install satysfi
$ cd doc
$ make
```

Then, an error `(Unix.Unix_error "Illegal seek" sysconf "")` occurred:

```sh-session
$ make
satysfi doc-primitives.saty -o primitives.pdf
Uncaught exception:
  
  (Unix.Unix_error "Illegal seek" sysconf "")

Raised by primitive operation at file "src/core_unix.ml", line 348, characters 14-29
Makefile:4: recipe for target 'all' failed
make: *** [all] Error 2
```

### Environment

* Ubuntu 17.10 (Artful Aardvark)
* Ubuntu GLIBC 2.26-0ubuntu2.1
* OCaml 4.05.0 (opam 1.2.2)
* SATySFi commit 3fb9591a442270d4a5f36f5c55726c2720e64eee

### How to fix

This bug is reported to janestreet/core#95. Updating core to v0.10.0 fixes the problem (currently, [`opam` config file](https://github.com/gfngfn/SATySFi/blob/3fb9591a442270d4a5f36f5c55726c2720e64eee/opam) pins its version to v0.9.1).

--- a/opam
+++ b/opam
@@ -27,7 +27,7 @@ depends: [
   "ocamlbuild" {build}
   "menhir"
   "ppx_deriving"
-  "core" {= "v0.9.1"}
+  "core" {= "v0.10.0"}
   "uutf"
   "result"
   "bitv"

--- a/src/backend/flowGraph.ml
+++ b/src/backend/flowGraph.ml
@@ -5,7 +5,7 @@ let print_for_debug msg =
 *)
   ()
 
-module Heap = Core.Heap.Removable
+module Heap = Core.Heap
 
 
 module type VertexType =

